doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,207 @@
+# Whoa, this can't be! The number of dividing nuclei
+
+print(labeled_dividing.max())
+
+#####################################################################
+# is overestimated, while the total number of cells
+
+print(labeled_cells.max())
+
+#####################################################################
+# is underestimated.
+
+fig, ax = plt.subplots(ncols=3, figsize=(15, 5))
+ax[0].imshow(image)
+ax[0].set_title('Original')
+ax[0].axis('off')
+ax[2].imshow(cells)
+ax[2].set_title('All nuclei?')
+ax[2].axis('off')
+ax[1].imshow(dividing)
+ax[1].set_title('Dividing nuclei?')
+ax[1].axis('off')
+plt.show()
+
+#####################################################################
+# Count dividing nuclei
+# =====================
+# Clearly, not all connected regions in the middle plot are dividing nuclei.
+# On one hand, the second threshold (value of ``thresholds[1]``) appears to be
+# too low to separate those very bright areas corresponding to dividing nuclei
+# from relatively bright pixels otherwise present in many nuclei. On the other
+# hand, we want a smoother image, removing small spurious objects and,
+# possibly, merging clusters of neighbouring objects (some could correspond to
+# two nuclei emerging from one cell division). In a way, the segmentation
+# challenge we are facing with dividing nuclei is the opposite of that with
+# (touching) cells.
+
+#####################################################################
+# To find suitable values for thresholds and filtering parameters, we proceed
+# by dichotomy, visually and manually.
+
+higher_threshold = 125
+dividing = image > higher_threshold
+
+smoother_dividing = filters.rank.mean(util.img_as_ubyte(dividing),
+                                      morphology.disk(4))
+
+binary_smoother_dividing = smoother_dividing > 20
+
+fig, ax = plt.subplots(figsize=(5, 5))
+ax.imshow(binary_smoother_dividing)
+ax.set_title('Dividing nuclei')
+ax.axis('off')
+plt.show()
+
+#####################################################################
+# We are left with
+cleaned_dividing = measure.label(binary_smoother_dividing)
+print(cleaned_dividing.max())
+
+#####################################################################
+# dividing nuclei in this sample.
+
+#####################################################################
+# Segment nuclei
+# ==============
+# To separate overlapping nuclei, we resort to
+# :ref:`sphx_glr_auto_examples_segmentation_plot_watershed.py`.
+# To visualize the segmentation conveniently, we colour-code the labelled
+# regions using the `color.label2rgb` function, specifying the background
+# label with argument `bg_label=0`.
+
+distance = ndi.distance_transform_edt(cells)
+
+local_maxi = feature.peak_local_max(distance, indices=False,
+                                    min_distance=7)
+
+markers = measure.label(local_maxi)
+
+segmented_cells = morphology.watershed(-distance, markers, mask=cells)
```

Comment:
please use `segmentation.watershed` to avoid the warning

Reply:
done 5339e62c293de6803116b1e08a8e0fa684779f04